draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -104,24 +104,20 @@
         <t>
            An offerer and answerer MUST, in each offer and answer, include an SDP 'dcmap' 
            attribute <xref target="I-D.ietf-mmusic-data-channel-sdpneg"/> in the SDP media
-           descripton (m= line) <xref target="RFC4566"/> describing the SCTP association
+           descripton (m= section) <xref target="RFC4566"/> describing the SCTP association
            <xref target="RFC4960"/> used to realize the T.140 data channel.
         </t>
         <t>
-          The offerer and answerer MUST include the following attribute parameters, and
-          parameter values in the dcmap attribute:
-          <list style="symbols">
-            <t>"label=" labelstring</t>
-            <t>"subprotocol=" "t140"</t>
-          </list>
+          The offerer and answerer MUST include the subprotocol attribute parameter,
+          with a "t140" parameter value, in the dcmap attribute value.
         </t>
         <t>
-          The offerer and answerer MUST NOT include the max-retr, max-time and ordered
-          attribute parameters in the dcmap attribute.
+          The offerer and answerer MAY, based on local policy, include the priority
+          attribute parameter and the label attribute parameter in the dcmap attribute value.
```

Comment:
Is "label" really included based on local policy? I think it is when needed by the protocol.  Can we delete the "based on local policy" for both, or do you need to reorder the wording?

Reply:
I can remove "based on local policy".